src/main/java/org/broadinstitute/hellbender/utils/spark/JoinStrategy.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.hellbender.utils.spark;
+
+/**
+ * Possible join strategies when using Spark
+ */
+public enum JoinStrategy {
```

Comment:
Think this would be better off under `engine.spark`, since preparing/joining inputs is an engine-level task.